Makefile

```diff
@@ -0,0 +1,13 @@
+PWD := $(shell pwd)
+GO_FILES := $(shell find . -type f -name "*.go" -print)
+
+k6: $(GO_FILES)
+	xk6 build \
+		--replace "github.com/mingrammer/flog=github.com/chaudum/flog@v0.4.4-0.20211115125504-92153be038e6" \
```

Comment:
`github.com/mingrammer/flog` is not importable, therefore the replacement with [`chaudum/flog`](https://github.com/chaudum/flog/commit/92153be038e65a9383ed50fdf9df1eac8a42fdc0).

Reply:
I will try to get the changes into the upstream project.

Reply:
We can fork this in the repo for sure.